akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/AbstractBehavior.scala

```diff
@@ -16,14 +16,20 @@ import akka.util.OptionVal
  * alternative is provided by the factory methods in [[Behaviors]], for example
  * [[Behaviors.receiveMessage]].
  *
- * Instances of this behavior should be created via [[Behaviors.setup]] and if
- * the [[ActorContext]] is needed it can be passed as a constructor parameter
- * from the factory function.  This is important because a new instance
+ * Instances of this behavior should be created via [[Behaviors.setup]] and
+ * the [[ActorContext]] should be passed as a constructor parameter
+ * from the factory function. This is important because a new instance
```

Comment:
Not sure the distinction is important to point out but, maybe something about the FSM use case where you create the behavior to return a new state, where you already have the context of the actor.  Hard to phrase concisely though.

Maybe also a "Must never be created with the context of another actor such as the parent or else it will fail on the first received message." (as per the safety guard below)

Reply:
Added some more clarifications of what is allowed and not: https://github.com/akka/akka/pull/27806/commits/1d785266d46570009672de2371df2b6326468f2f